pom.xml

```diff
@@ -113,6 +113,16 @@
                 </exclusion>
             </exclusions>
         </dependency>
```

Comment:
Any reason why the <DependencyManagement> tag isn't included?

Reply:
I did not include the DependencyManagement tag because it results in test errors if I do so The test errors say they cannot find the configurations. This might possibly be because eva-pipeline's POM already has a parent POM - for reference: [Spring Boot Dependency Management with a Custom Parent](https://www.baeldung.com/spring-boot-dependency-management-custom-parent).

Also, Travis CLI build passes without the tag. So the other reason was to keep the eva-pipeline POM lean. 

To test if I could add dependencyManagement, I added the following to eva-pipeline's POM, which results in test errors (because it is unable to find Spring configurations). 
I'm new to Maven so I'm not 100% sure about my reasons. So please find the code used to include the tag:
```
    <dependencyManagement>
        <dependencies>
            <dependency>
                <groupId>uk.ac.ebi.eva</groupId>
                <artifactId>variation-commons</artifactId>
                <version>0.7-SNAPSHOT</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
        </dependencies>
    </dependencyManagement>
```

Errors received were like so: 
```
java.lang.NoSuchMethodError: org.springframework.util.ClassUtils.getQualifiedMethodName(Ljava/lang/reflect/Method;Ljava/lang/Class;)Ljava/lang/String;
        at org.springframework.transaction.interceptor.TransactionAspectSupport.methodIdentification(TransactionAspectSupport.java:396)
```

Reply:
Hi, please have a look at this comment thread. 

My current understanding (something I haven't tried yet) is to include dependencies via a singular variation-commons. 

So for example, only the following tag should suffice:
<dependency>
    <groupid>uk.ac.eba.eva</groupid>
    <artifactid>variation-commons</artifactid>
    <version>0.7-SNAPSHOT</version>
</dependency>